webhooks/webhook.go

```diff
@@ -1,4 +1,12 @@
 package webhooks
 
+import (
+	"net/http"
+)
+
 const WebhookContextPullRequest = "changelog/pull-request"
 const WebhookContextPush = "changelog/push"
+
+type VCSWebhook interface {
+	New(string, string) http.Handler
```

Comment:
Can you put parameter names on this interface? Its not immediately obvious what each param is